test/fixtures/hdfs-fixture/build.gradle

```diff
@@ -13,8 +13,29 @@ apply plugin: 'com.gradleup.shadow'
 
 import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
 
+def patched = Attribute.of('patched', Boolean)
+dependencies {
```

Comment:
nit. the usual order in a build script is configuration definitions before dependencies. kind of best practice we usually follow. so ideally we would put that dependencies{} block after the configurations{} block